neurom/core/tests/test_tree.py

```diff
@@ -393,3 +377,23 @@ def _build_tuple(tree):
 
     nt.assert_equal(_build_tuple(REF_TREE2.children[0].children[0].children[0]),
                     (11111, 11112, 11113))
+
+
+def test_make_copy():
```

Comment:
We need a test that shows that modifying a copied tree does not modify the original